<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditorDialog</class>
 <widget class="QDialog" name="ItemEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblPaymentType">
       <property name="text">
        <string>Тип оплаты</string>
       </property>
       <property name="buddy">
        <cstring>cmbPaymentType</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbPaymentType">
       <property name="minimumSize">
        <size>
         <width>234</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>По безналу</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>По протоколу</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblCodePart">
       <property name="text">
        <string>Часть кода</string>
       </property>
       <property name="buddy">
        <cstring>cmbCodePart</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cmbCodePart">
       <property name="minimumSize">
        <size>
         <width>234</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblOKPF">
       <property name="text">
        <string>Организационная форма</string>
       </property>
       <property name="buddy">
        <cstring>cmbOKPF</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="CRBComboBox" name="cmbOKPF">
       <property name="minimumSize">
        <size>
         <width>234</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblOrgName">
       <property name="text">
        <string>Наименование организации</string>
       </property>
       <property name="buddy">
        <cstring>cmbOrgId</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="COrgComboBox" name="cmbOrgId">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnSelectOrgName">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblContractNumber">
       <property name="text">
        <string>Номер договора</string>
       </property>
       <property name="buddy">
        <cstring>edtContractNumber</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="edtContractNumber">
       <property name="minimumSize">
        <size>
         <width>234</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblDate">
       <property name="text">
        <string>Дата заключения договора</string>
       </property>
       <property name="buddy">
        <cstring>edtDate</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="CDateEdit" name="edtDate"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblEndDate">
       <property name="text">
        <string>Дата окончания договора</string>
       </property>
       <property name="buddy">
        <cstring>edtEndDate</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="CDateEdit" name="edtEndDate"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lblContractSum">
       <property name="text">
        <string>Сумма договора</string>
       </property>
       <property name="buddy">
        <cstring>edtContractSum</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="edtContractSum">
       <property name="minimumSize">
        <size>
         <width>234</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="lblContragent">
       <property name="text">
        <string>Контрагент</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="edtContragent"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="lblProtocolNumber">
       <property name="text">
        <string>Номер протокола</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="edtProtocolNum"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="lblProtocolPhase">
       <property name="text">
        <string>Фаза протокола</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="edtProtocolPhase"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="lblProtocolName">
       <property name="text">
        <string>Наименование протокола</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="edtProtocolName"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="lblRecPatients">
       <property name="text">
        <string>Набор пациентов</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QComboBox" name="cmbRecPatients">
       <item>
        <property name="text">
         <string>Набор открыт</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Набор закрыт</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="lblContractItem">
       <property name="text">
        <string>Предмет договора</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLineEdit" name="edtContractItem"/>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="lblAddress">
       <property name="text">
        <string>Почтовый адрес</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLineEdit" name="edtAddress"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
